papers/Mathematics/paper.Rmd

```diff
@@ -0,0 +1,269 @@
+library(see)
+
+plot(outliers)
+```
+
+Other univariate methods are available, such as using the interquartile range (IQR), or based on different intervals, such as the Highest Density Interval (HDI) or the Bias Corrected and Accelerated Interval (BCI). These methods are documented and described in the function's [help page](<https://easystats.github.io/performance/reference/check_outliers.html>).
+
+## Multivariate Outliers
+
+Univariate outliers can be useful when the focus is on a particular variable, for instance the reaction time, as extreme values might be indicative of inattention or non-task-related behavior^[ Note that they might not be the optimal way of treating reaction time outliers [@ratcliff1993methods; @van1995statistical]].
+
+However, in many scenarios, variables of a data set are not independent, and an abnormal observation will impact multiple dimensions. For instance, a participant giving random answers to a questionnaire. In this case, computing the _z_ score for each of the questions might not lead to satisfactory results. Instead, one might want to look at these variables together.
+
+One common approach for this is to compute multivariate distance metrics such as the Mahalanobis distance. Although the Mahalanobis distance is very popular, just like the regular _z_ scores method, it is not robust and is heavily influenced by the outliers themselves. Therefore, for multivariate outliers, it is recommended to use the Minimum Covariance Determinant, a robust version of the Mahalanobis distance [MCD, @leys2018outliers; @leys2019outliers]. 
+
+In *{performance}*'s `check_outliers()`, one can use this approach with `method = "mcd"`.^[Our default threshold for the MCD method is defined by `stats::qchisq(p = 1 - 0.001, df = ncol(x))`, which again is an approximation of the critical value for _p_ < .001 consistent with the thresholds of our other methods.]
+
+```{r multivariate, fig.cap = "Visual depiction of outliers using the Minimum Covariance Determinant (MCD) method, a robust version of the Mahalanobis distance."}
+results <- check_outliers(data, method = "mcd")
+results
+
+plot(results)
+```
+
+Other multivariate methods are available, such as another type of robust Mahalanobis distance that in this case relies on an orthogonalized Gnanadesikan-Kettenring pairwise estimator [@gnanadesikan1972robust]. These methods are documented and described in the function's [help page](https://easystats.github.io/performance/reference/check_outliers.html).
+
+## Model-Based Outliers
+
+Working with regression models creates the possibility of using model-based SOD methods. These methods rely on the concept of *leverage*, that is, how much influence a given observation can have on the whole model. If few observations have a relatively strong leverage/influence on the model, one can suspect that the model's estimates are biased by these observations, in which case flagging them as outliers could prove helpful.
+
```

Comment:
> "can have on the whole model" - what does this mean? Are only estimates affected, or model fit indices as well? So, what is my strongest concern here?

Not sure. That was my paraphrasing of Mattan's

> Something something... what is leverage... why we should care if few observations have (relatively) strong leverage (answer: they are suspect of biasing our estimates!).

I guess I'll use model estimates instead of just model (was trying to simplify for the public but I guess "model" is too vague). I think it would make sense that it affects model fit indices too, but I can't really comment on that though since @mattansb didn't include that in his first sentence. Unless he confirms a better phrasing :)

> "in which case flagging them as outliers could prove helpful." - in which way? what would be the next step where I need this help?

I think this is covered in the next section, "Handling Outliers". But I guess I can a parenthesis, e.g., (see next section, "Handling Outliers").

Reply:
Sounds good to me